src/fixtures/config.js

```diff
@@ -56,6 +56,7 @@ is_government: false
 shareicon: /images/isomer-logo.svg
 facebook-pixel: "true"
 google_analytics: UA-39345131-33
+google_analytics_ga4: GA-test
```

Comment:
where will the test value be replaced with an actual value?

Reply:
We never actually test this with actual values - since this is only retrieved from github, and we mock github calls in all our tests, none of our backend tests would use an actual value of this, and the other values here are dummy values taken from our default template params iirc

We only check for actual values along with the settings e2e tests on the frontend, but those tests are currently not working properly